tests/Application/config/bundles.php

```diff
@@ -56,4 +56,6 @@
     SitemapPlugin\SitemapPlugin::class => ['all' => true],
     Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true, 'test' => true, 'test_cached' => true],
     Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true, 'test_cached' => true],
+    Nelmio\Alice\Bridge\Symfony\NelmioAliceBundle::class => ['all' => true],
```

Comment:
Doesn't matter in a test application, but if people go through your test application to see what you've done, some will possibly think that they need to add these two bundles for all environments, but I think it should be:

```suggestion
    Nelmio\Alice\Bridge\Symfony\NelmioAliceBundle::class => ['dev' => true, 'test' => true, 'test_cached' => true],
```